.golangci.yml

```diff
@@ -1,6 +1,79 @@
----
+linters-settings:
+  maligned:
+    suggest-new: true
+  misspell:
+    locale: US
+
 issues:
   exclude-rules:
-  - path: _test.go
-    linters:
+    - path: _test\.go
+      linters:
+        - dupl
+        - goerr113
+
+linters:
+  disable-all: true
+  enable:
+    - asciicheck
+    - bodyclose
+    # - cyclop - enable later
+    - deadcode
```

Comment:
`deadcode` is enabled by default (see [here](https://golangci-lint.run/usage/linters/)). If I remove `disable-all`, it will always be enabled then 😄 Instructions unclear 🤖 

As are `structcheck`, `unused` and `varcheck` too.

Reply:
Let's explicitly disable the deprecated linters.

Something like this:

```yaml
linters:
  enable:
    - revive
  disable:
    # Disable soon to deprecated[1] linters that lead to false
    # positives when build tags disable certain files[2]
    # 1: https://github.com/golangci/golangci-lint/issues/1841
    # 2: https://github.com/prometheus/node_exporter/issues/1545
    - deadcode
    - unused
    - structcheck
    - varcheck
```

(This is what I do in prometheus/node_exporter)